packages/chart/src/components/BarChart/components/BarChart.Horizontal.tsx

```diff
@@ -149,7 +149,7 @@ export const BarChartHorizontal = () => {
                     return barColor
                   }
 
-                  const iconPadding = symbol => (symbol === 'Asterisk' ? '3px' : symbol === 'Double Asterisks' ? '4px' : '12px')
+                  const iconPadding = 12 + suppressedBarWidth
```

Comment:
Use a variable for 12 to describe where 12 came from.

Reply:
Fixed. 12 was just padding to add space. Hard coded.